ush/templates/FV3LAM_wflow.xml

```diff
@@ -2251,6 +2331,9 @@ MODULES_RUN_TASK_FP script.
        <and>
          <timedep><cyclestr offset="&START_TIME_CONVENTIONAL;">@Y@m@d@H@M00</cyclestr></timedep>
          <taskdep task="&PREP_CYC_TN;_prod{{ uscore_ensmem_name }}"/>
+         {%- if do_sddacycle %}
+         <taskdep task="&ANAL_SD_GSI_TN;_prod"/>
+         {%- endif %}
```

Comment:
PM analysis has to run at the same time as conventional DA. This dependence should not be here.